frontend/src/RecordingPageComp/RecordingSelectionComp.jsx

```diff
@@ -33,6 +36,13 @@ function RecordingSelector({ setSelectedBlob, selectedBlob }) {
         closestElem = elem;
         minDistance = distance;
       }
+
```

Comment:
No need to add comments like this as the code is self-explanatory. Make sure to use discretion when adding comments. Too many comments can create clutter. I'd suggest taking a pass through the code and eliminating any unnecessary comments. Here are some best practices on writing comments in code:
https://stackoverflow.blog/2021/12/23/best-practices-for-writing-code-comments/